src/applications/personalization/profile-2/components/Profile2Wrapper.jsx

```diff
@@ -19,12 +19,12 @@ import {
   fetchPersonalInformation as fetchPersonalInformationAction,
 } from 'applications/personalization/profile360/actions';
 import { fetchPaymentInformation as fetchPaymentInformationAction } from 'applications/personalization/profile360/actions/paymentInformation';
-
 import ProfileHeader from './ProfileHeader';
 import ProfileSideNav from './ProfileSideNav';
 import MobileMenuTrigger from './MobileMenuTrigger';
+import routes from '../routes';
 
-class ProfileWrapper extends Component {
+class Profile2 extends Component {
```

Comment:
Typo here, I think this should this be `<ProfileSupreme/>`

Reply:
How did I miss that.. :no_mouth: Shame on me!